pages/common/glab-auth.md

```diff
@@ -0,0 +1,20 @@
+# glab auth
+
+> Authenticate with a GitLab host from the command-line.
+> More information: <https://glab.readthedocs.io/en/latest/auth/index.html#synopsis>.
+
+- Log in with interactive prompt:
+
+`glab auth login`
+
+- Log in with a token from standard input (created in https://gitlab.com/-/profile/personal_access_tokens):
+
+`echo {{your_token}} | glab auth login --stdin`
```

Comment:
I think instead of piping we should use the `--token`
```suggestion
`glab auth login --token={{your_token}}`
```

Reply:
Pipes were also used in the documentation of `gh auth`.

Reply:
Sure, I have nothing against pipes per se :)

Reply:
the `--token` argument is better.